browser/mapping.go

```diff
@@ -130,25 +130,50 @@ func mapResponse(ctx context.Context, vu k6modules.VU, r api.Response) mapping {
 	return maps
 }
 
+// mapJSHandle to the JS module.
+func mapJSHandle(ctx context.Context, vu k6modules.VU, eh api.JSHandle) mapping {
+	rt := vu.Runtime()
+	return mapping{
+		"asElement": func() *goja.Object {
+			m := mapElementHandle(ctx, vu, eh.AsElement())
+			return rt.ToValue(m).ToObject(rt)
+		},
+		"dispose":  eh.Dispose,
+		"evaluate": eh.Evaluate,
+		"evaluateHandle": func(pageFunc goja.Value, args ...goja.Value) *goja.Object {
+			var (
+				h = eh.EvaluateHandle(pageFunc, args...)
+				m = mapJSHandle(ctx, vu, h)
+			)
+			return rt.ToValue(m).ToObject(rt)
+		},
+		"getProperties": func() *goja.Object {
+			dst := make(map[string]any)
+			for k, v := range eh.GetProperties() {
+				dst[k] = mapJSHandle(ctx, vu, v)
+			}
+			return rt.ToValue(dst).ToObject(rt)
+		},
+		"getProperty": func(propertyName string) *goja.Object {
+			var (
+				h = eh.GetProperty(propertyName)
+				m = mapJSHandle(ctx, vu, h)
+			)
+			return rt.ToValue(m).ToObject(rt)
+		},
+		"jSONValue": eh.JSONValue,
```

Comment:
This should be `jsonValue`, no?

Reference: https://playwright.dev/docs/api/class-jshandle#js-handle-json-value

Reply:
Resolved [here](https://github.com/grafana/xk6-browser/blob/d50f4ba1a7934371c122116fdcf2c8653618a08d/browser/mapping.go#L164).